### PR TITLE
Split homework

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/course_materials/exercises/05_Control_structures/homework_while.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Control Structures - Homework exercise
```
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_while.md:1]: done

File: docs/course_materials/exercises/05_Control_structures/homework_while.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Control Structures - Homework exercise 
+
+## while loop
+
+### while_average.py
+Create a file *while_average.py*
+
```
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_while.md:7]: done

File: docs/course_materials/exercises/05_Control_structures/homework_while.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+for example: 
+
+the user input is : 15, 16, 12, 0
+The output should be: 
+Sum of 15 + 16  + 12 = 43 
+Average of the numbers = 14.3 
+
+### positive_negative.py
+Create a file *positive_negative.py*
+
```
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_while.md:21]: done.

File: docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md [hunk 4]
```diff
@@ -3,12 +3,13 @@
 ## Variables, numbers and boolean
```
Comment (shiramax) [reply docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md:3]: done

**Review by sleviim — APPROVED** [review]

File: docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md (hunk 4 above)
Comment (sleviim) [inline docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md:3]: Please add  - HomeWork at the end of the line

File: docs/course_materials/exercises/05_Control_structures/homework_if.md [hunk 5]
```diff
@@ -4,62 +4,33 @@
 ## if & else
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_if.md:4]: Please add  - HomeWork  at the end of the line

File: docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md (hunk 4 above)
Comment (sleviim) [inline docs/course_materials/exercises/03_Variables_and_datatypes/homework_variables.md:3]: Please add - HomeWork at the end of the line

File: docs/course_materials/exercises/05_Control_structures/homework_if.md [hunk 6]
```diff
@@ -4,62 +4,33 @@
...
 
 ### even_or_odd.py
+Create a file *even_or_odd.py*
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_if.md:7]: Replace the order of the lines:
new line
Create a file...

File: docs/course_materials/exercises/05_Control_structures/homework_if.md [hunk 7]
```diff
@@ -4,62 +4,33 @@
...
+
 Use 'input' to get a number from the user.
 Print "Even" if the number is even (divided by 2 without reminder) and "Odd" if
 the number is not even.
 Hint: a = 22 a % 2 == 0 #(Even!!) True
 
 ### greater_then.py
+Create a file *greater_then.py*
+
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_if.md:16]: same

File: docs/course_materials/exercises/05_Control_structures/homework_if.md [hunk 8]
```diff
@@ -4,62 +4,33 @@
...
 Write a program that accepts a number X from the user and prints:
 ● "The number is greater than 1000" if the number is greater than 1000 
 ● otherwise "The number is less than 1000"
 
 ### buy_2_get_3.py
+Create a file *buy_2_get_3.py*
+
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_if.md:23]: same

File: docs/course_materials/exercises/05_Control_structures/homework_if.md [hunk 9]
```diff
@@ -4,62 +4,33 @@
...
 Write a program that checks how many items you have purchased in an online store, 
 The program receives from the client amount of items you would like to purchase
 if it's 2 print a message you can another item free!
 if it's 1 print a message for the client that it pays to buy another item on the site because he will get another item for free
     
 ### bill.py
+Create a file *bill.py*
+
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_if.md:31]: same

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
+# Control Structures - Homework exercise
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_loop.md:1]: # Control Structures - Homework exercise  > # Control Structures - HomeWork

File: docs/course_materials/exercises/05_Control_structures/homework_while.md (hunk 1 above)
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_while.md:1]: same

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
+# Control Structures - Homework exercise 
+
+## for loop
+
+### ten.py
+Create a file *ten.py*
+
+Write a program to print numbers from 1 to 10.
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_loop.md:8]: Write a program to print numbers from 1 to 10. > Write a program that prints the numbers from 1 to 10.

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+### natural.py
+Create a file *natural.py*
+
+Write a program to calculate the sum of first 10 natural number.
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_loop.md:13]: Write a program to calculate the sum of first 10 natural number > 
Write a program that calculates the sum of the first 10 natural numbers

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+## for loop
+
+### ten.py
+Create a file *ten.py*
+
+Write a program to print numbers from 1 to 10.
+
+### natural.py
+Create a file *natural.py*
+
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_loop.md:12]: Replace the order of the lines:
new line
Create a file...

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
+# Control Structures - Homework exercise 
+
+## for loop
+
+### ten.py
+Create a file *ten.py*
+
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_loop.md:7]: Replace the order of the lines:
new line
Create a file...

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+Write a program to print numbers from 1 to 10.
+
+### natural.py
+Create a file *natural.py*
+
+Write a program to calculate the sum of first 10 natural number.
+
+### divided_by_5.py
+Create a file *divided_by_5.py*
+
```
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_loop.md:17]: same

File: docs/course_materials/exercises/05_Control_structures/homework_while.md (hunk 2 above)
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_while.md:7]: same

File: docs/course_materials/exercises/05_Control_structures/homework_while.md (hunk 3 above)
Comment (sleviim) [inline docs/course_materials/exercises/05_Control_structures/homework_while.md:21]: same

**Review by sleviim — CHANGES_REQUESTED** [review]

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md (hunk 11 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_loop.md:8]: done.

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md (hunk 14 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_loop.md:7]: done.

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md (hunk 10 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_loop.md:1]: done

File: docs/course_materials/exercises/05_Control_structures/homework_if.md (hunk 9 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_if.md:31]: done.

File: docs/course_materials/exercises/05_Control_structures/homework_if.md (hunk 8 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_if.md:23]: done

File: docs/course_materials/exercises/05_Control_structures/homework_if.md (hunk 7 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_if.md:16]: done

File: docs/course_materials/exercises/05_Control_structures/homework_if.md (hunk 6 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_if.md:7]: done.

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md (hunk 13 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_loop.md:12]: done

File: docs/course_materials/exercises/05_Control_structures/homework_loop.md (hunk 15 above)
Comment (shiramax) [reply docs/course_materials/exercises/05_Control_structures/homework_loop.md:17]: done

**Pull request merged**